mlflow/utils/databricks_utils.py

```diff
@@ -261,6 +261,18 @@ def get_job_type():
         return _get_context_tag("jobTaskType")
 
 
+@_use_repl_context_if_available("jobType")
+def get_job_type_info():
+    try:
+        return _get_context_tag("jobType")
+    except Exception:
+        return None
+
+
+def get_experiment_name_from_job_id(job_id):
+    return "job:/" + job_id
```

Comment:
Let's make sure this matches what we discussed

Reply:
Per Slack discussion, this is what we want:)